### PR TITLE
PYIC-7333: add missing scenarios for f2f journey

### PR DESCRIPTION
**Pull request by shivanshuit914**
## Proposed changes

- Add F2f requested strength score API tests

### What changed

- Added f2f API tests to cover requested strength score  scenario

### Issue tracking

- [PYIC-7333](https://govukverify.atlassian.net/browse/PYIC-7333)

[PYIC-7333]: https://govukverify.atlassian.net/browse/PYIC-7333?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api-tests/features/p2-f2f-journey.feature [hunk 1]
```diff
@@ -138,3 +138,68 @@ Feature: P2 F2F journey
...
+    Scenario Outline: Async queue error
+      When I get an error from the async CRI stub
+      Then I get a 'page-face-to-face-handoff' page response
+
+      # Return journey
+      When I start a new 'medium-confidence' journey and return to a 'pyi-f2f-technical' page response
+      When I submit a 'end' event
+      Then I get an OAuth response
+
+  Scenario: f2f PYI escpae route
```
Comment (Joe-Edwards-GDS) [inline api-tests/features/p2-f2f-journey.feature]: Typo: escape

(Where possible I think we should also try and be consistent about capitalising F2F in text)

File: api-tests/features/p2-f2f-journey.feature [hunk 2]
```diff
@@ -138,3 +138,68 @@ Feature: P2 F2F journey
...
+    Scenario Outline: requested strengh score four for fraud score 1
+      When I submit 'kenneth-score-1' details to the CRI stub
+      Then I get a 'f2f' CRI response
+      When I submit 'kenneth-passport-valid' details with attributes to the CRI stub
+        | Attribute          | Values                                          |
+        | evidence_requested | {"scoringPolicy":"gpg45","strengthScore":4} |
+      Then I get a 'page-face-to-face-handoff' page response
+
+    Scenario Outline: requested strengh score four for history score 0
+      When I submit 'kenneth-score-1-history-0' details to the CRI stub
```
Comment (Joe-Edwards-GDS) [inline api-tests/features/p2-f2f-journey.feature:192]: This is a bit of a strange scenario - this _also_ has fraud score 1 (which forces strength 4).

TBH, we already have unit tests for this detailed logic, so checking the cases we expect to see in real life (fraud 1 vs. 2) is the important thing.

File: api-tests/features/p2-f2f-journey.feature [hunk 3]
```diff
@@ -138,3 +138,68 @@ Feature: P2 F2F journey
...
+      When I submit a 'end' event
+      Then I get a 'pyi-post-office' page response
+      When I submit a 'next' event
+      Then I get a 'claimedIdentity' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+
+    Scenario Outline: requested strengh score three for fraud score 2
```
Comment (Joe-Edwards-GDS) [inline api-tests/features/p2-f2f-journey.feature]: Typo: strength

(and below)

File: api-tests/features/p2-f2f-journey.feature [hunk 4]
```diff
@@ -138,3 +138,68 @@ Feature: P2 F2F journey
...
+
+    Scenario Outline: requested strengh score four for history score 0
+      When I submit 'kenneth-score-1-history-0' details to the CRI stub
+      Then I get a 'f2f' CRI response
+      When I submit 'kenneth-passport-valid' details with attributes to the CRI stub
+        | Attribute          | Values                                          |
+        | evidence_requested | {"scoringPolicy":"gpg45","strengthScore":4} |
+      Then I get a 'page-face-to-face-handoff' page response
+
+    Scenario Outline: requested strengh score four for history score 1
```
Comment (Joe-Edwards-GDS) [inline api-tests/features/p2-f2f-journey.feature]: This says history score 1, but the details used have activity history 0?

File: api-tests/features/p2-f2f-journey.feature (hunk 3 above)
Comment (Joe-Edwards-GDS) [inline api-tests/features/p2-f2f-journey.feature]: Why are these `Scenario Outline`s - if they're not parameterised, should they just be `Scenario`s?

(Alternatively, they are quite repetitive and maybe could be combined a single outline)

**Review by Joe-Edwards-GDS — COMMENTED** [review]
Looks reasonable, but a few typos/comments

File: api-tests/features/p2-f2f-journey.feature (hunk 4 above)
Comment (shivanshuit914) [reply api-tests/features/p2-f2f-journey.feature]: aah copy paste mistake :) thanks

**Review by Joe-Edwards-GDS — APPROVED** [review]

**Pull request merged**